### PR TITLE
Make the path relative to the github org and repo

### PR DESCRIPTION
**Pull request by asmith-nhsx**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <script>
     var App = angular.module('App', ['ngAria']);
     App.controller('ReposController', function ($scope, $http) {
-      $http.get('https://raw.githubusercontent.com/UKHomeOffice/xgov-opensource-repo-scraper/gh-pages/repos.json')
+      $http.get('https://raw.githubusercontent.com/'+window.location.href.substr(8,window.location.href.indexOf('.')-8)+window.location.pathname +'gh-pages/repos.json')
```
Comment (chrisns) [inline public/index.html]: does this not work?
```suggestion
      $http.get('./repos.json')
```

**Review by chrisns — CHANGES_REQUESTED** [review]

**Comment by asmith-nhsx** [comment]
Yes the simple relative path works, I was thinking that the "raw.githubusercontent.com" in the original code was important in someway...

**Review by chrisns — APPROVED** [review]
yeah, no idea why i did that!

**Comment by chrisns** [comment]
thanks @asmith-nhsx !

**Pull request merged**